main.tf

```diff
@@ -19,20 +19,22 @@
  *****************************************/
 locals {}
 
-resource "google_bigquery_dataset" "default" {
-  dataset_id                  = "${var.dataset_id}"
-  friendly_name               = "${var.dataset_name}"
-  description                 = "${var.description}"
+resource "google_bigquery_dataset" "main" {
+  dataset_id    = "${var.dataset_id}"
+  friendly_name = "${var.dataset_name}"
+  description   = "${var.description}"
+
   #TODO: add if condition to validate if neither US or EU are supplied
-  location                    = "${var.region}"
+  location = "${var.region}"
+
   #TODO: format this ne excluded by default but can optionally be defined if the user wishes
   default_table_expiration_ms = "${var.expiration}"
   project                     = "${var.project_id}"
 
   #TODO: Need to find a way to dynamically assign a dict object(s)
   labels {
-    env = "default"
-    foo = "bar"
+    env   = "default"
+    foo   = "bar"
     tonyd = "tonyd"
```

Comment:
As per our convention, labels should be placed in variables.tf file (https://docs.google.com/document/d/1lpuK7salw2gUPdpQByPFAS4IvmM2aeHE5eFx5W_WTd8/edit#bookmark=id.chv93ewt58zr)

Reply:
Also should probably remove the "tonyd" kv pair.

Reply:
noted, this right now is placeholder and will parameterize this in this next pass